selenium-system-test/src/test/resources/features/SCR personal detail validation .feature

```diff
@@ -0,0 +1,65 @@
+@test
+Feature: SCR personal detail validation
+
+  Scenario: Internal operator manually creates account on behalf of eligible SCR applicant
+    Given the operator is logged in with SCR clearance
+    And the SCR applicant is eligible
+    When the operator fills in the applicant's details
+    And check the applicant's details are correct
```

Comment:
grammatically speaking `check` should be `checks`, but I think `check` is the wrong word here anyway. `verifies` would be more exact here as you can check the details but find that they are wrong